models/solution/stop_info.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright © Mapotempo, 2021
+#
+# This file is part of Mapotempo.
+#
+# Mapotempo is free software. You can redistribute it and/or
+# modify since you respect the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# Mapotempo is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the Licenses for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Mapotempo. If not, see:
+# <http://www.gnu.org/licenses/agpl.html>
+#
+require './models/base'
+
+module Models
+  class Solution < Base
+    class Stop < Base
+      class Info < Base
+        field :id, as_json: :none
+        field :day_week_num
+        field :day_week
+
+        field :travel_distance
+        field :travel_time
+        field :travel_value
+
+        field :waiting_time, default: 0
```

Comment:
how can we make this more automatic, do we really need this field in the class, maybe we can calculated it with the info we have once at the very end (eventually inside the grape entity)?

Reply:
And maybe we can get rid of the "parse" logic this way/..

Reply:
The internal ticket 897